plutus-tx-plugin/test/Budget/Spec.hs

```diff
@@ -141,11 +165,51 @@ compiledFilter = $$(compile [||
       let ls = [1,2,3,4,5,6,7,8,9,10] :: [Integer]
        in PlutusTx.filter PlutusTx.even ls ||])
 
-compiledElem :: CompiledCode Bool
-compiledElem = $$(compile [||
+compiledAndCheap :: CompiledCode Bool
```

Comment:
TBH these could do with comments. What are they about? It's just a big list of random vairants - what are they showing?